tests/e2e/config/bootstrap.js

```diff
@@ -0,0 +1,185 @@
+ *
+ * @type {Object<string,string>}
+ */
+const OBSERVED_CONSOLE_MESSAGE_TYPES = {
+  warning: 'warn',
+  error: 'error',
+};
+
+/**
+ * Array of page event tuples of [ eventName, handler ].
+ *
+ * @type {Array}
+ */
+const pageEvents = [];
+
+// The Jest timeout is increased because these tests are a bit slow
+jest.setTimeout(PUPPETEER_TIMEOUT || 100000);
+
+// Set default timeout for individual expect-puppeteer assertions. (Default: 500)
+setDefaultOptions({ timeout: EXPECT_PUPPETEER_TIMEOUT || 500 });
+
+/**
+ * Adds an event listener to the page to handle additions of page event
+ * handlers, to assure that they are removed at test teardown.
+ */
+function capturePageEventsForTearDown() {
+  page.on('newListener', (eventName, listener) => {
+    pageEvents.push([eventName, listener]);
+  });
+}
+
+/**
+ * Removes all bound page event handlers.
+ */
+function removePageEvents() {
+  pageEvents.forEach(([eventName, handler]) => {
+    page.removeListener(eventName, handler);
+  });
+}
+
+/**
+ * Adds a page event handler to emit uncaught exception to process if one of
+ * the observed console logging types is encountered.
+ */
+function observeConsoleLogging() {
+  page.on('console', (message) => {
+    const type = message.type();
+    if (!OBSERVED_CONSOLE_MESSAGE_TYPES.hasOwnProperty(type)) {
+      return;
+    }
+
+    let text = message.text();
+
+    // styled-components warns about dynamically created components.
+    // @todo Fix issues.
+    if (text.includes(' has been created dynamically.')) {
+      return;
+    }
+
+    const logFunction = OBSERVED_CONSOLE_MESSAGE_TYPES[type];
+
+    // As of Puppeteer 1.6.1, `message.text()` wrongly returns an object of
+    // type JSHandle for error logging, instead of the expected string.
+    //
+    // See: https://github.com/GoogleChrome/puppeteer/issues/3397
+    //
+    // The recommendation there to asynchronously resolve the error value
+    // upon a console event may be prone to a race condition with the test
+    // completion, leaving a possibility of an error not being surfaced
+    // correctly. Instead, the logic here synchronously inspects the
+    // internal object shape of the JSHandle to find the error text. If it
+    // cannot be found, the default text value is used instead.
+    text = message.args()?.[0]?._remoteObject?.description || text;
+
+    // Disable reason: We intentionally bubble up the console message
+    // which, unless the test explicitly anticipates the logging via
+    // @wordpress/jest-console matchers, will cause the intended test
+    // failure.
+
+    // eslint-disable-next-line no-console
+    console[logFunction](text);
+  });
+}
+
+/**
+ * Runs Axe tests when the story editor is found on the current page.
+ *
+ * @return {?Promise} Promise resolving once Axe texts are finished.
+ */
+async function runAxeTestsForStoriesEditor() {
+  if (!(await page.$('body.edit-story'))) {
+    return;
+  }
+
+  await expect(page).toPassAxeTests({
```

Comment:
Why do we need to disabled these rules? When I put these in, it was because the old editor didn't support them.

Reply:
Ours also doesn't support the rules listed here yet.

Note that this list differs from  the one in the legacy stories editor.

I basically ran the tests against the current editor and disabled all failing tests for now.